rpc/context.go

```diff
@@ -73,6 +74,20 @@ func ContextAuthMetadata(ctx context.Context) map[string]string {
 	return authMD.(map[string]string)
 }
 
+// contextWithAllAuthClaims attaches authentication jwt claims to the given context.
+func contextWithAllAuthClaims(ctx context.Context, claims Claims) context.Context {
+	return context.WithValue(ctx, ctxKeyAuthClaims, claims)
+}
+
+// ContextAllAuthClaims returns authentication jwt claims.
+func ContextAllAuthClaims(ctx context.Context) Claims {
```

Comment:
[nit] I think ContextAuthClaims is cleaner

Reply:
Works for me. Updated